ffi/diplomat/c/examples/fixeddecimal_tiny/icu_capi_staticlib_tiny/Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = "2021"
 crate-type = ["staticlib"]
 
 [dependencies]
-icu_capi = { path = "../../../..", default-features = false, features = ["buffer_provider"] }
+icu_capi = { path = "../../../..", default-features = false, features = ["buffer_provider", "default_components"] }
```

Comment:
Suggestion: utilize this opportunity to test the individual component features. I think you need only `icu_decimal`.